docs/projects/ad9656_fmc/index.rst

```diff
@@ -0,0 +1,258 @@
+The :adi:`AD9656` has an internal clock divider (ratios 1-8) for higher
+frequency clocks.
+
+There are two ways to configure the clock source for :adi:`AD9656`:
+
+#. Default Configuration (On-board Oscillator):
+
+   - The board uses a 125MHz crystal oscillator (Y801) through a
+     transformer-coupled input network.
+
+#. Using an External Clock Source:
+
+   - Remove C302 (optional) and Jumper J304: This disables the on-board
+     oscillator.
+   - Attach the external clock source to the SMA connector labeled
+     CLOCK+ (J302). The external clock should be a clean signal generator,
+     typically ~2.8V p-p or 13 dBm sine wave input.
+
+For more details, check :adi:`EVAL-AD9656` schematic.
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+Check-out the table below to find out the conditions.
+
+================== ===============
+Instance           Zynq/Microblaze
+================== ===============
+rx_ad9656_tpl_core 0x44A00000
+axi_ad9656_rx_xcvr 0x44A60000
+axi_ad9656_rx_jesd 0x44AA0000
+axi_ad9656_rx_dma  0x7C400000
+================== ===============
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD9656
+     - 1
+   * - PS
+     - SPI 0
+     - AD9508
+     - 0
+   * - PS
+     - SPI 0
+     - AD9953
+     - 1
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+.. list-table::
+   :widths: 30 10 15 15
+   :header-rows: 1
+
+   * - Instance name
+     - HDL
+     - Linux ZynqMP
+     - Actual ZynqMP
+   * - axi_ad9656_rx_jesd
+     - 12
+     - 108
+     - 140
+   * - axi_ad9656_rx_dma
+     - 13
+     - 109
+     - 141
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI distributes the bit/elf files of these projects as part of the
+:dokuwiki:`ADI Kuiper Linux <resources/tools-software/linux-software/kuiper-linux>`.
+If you want to build the sources, ADI makes them available on the
+:git-hdl:`HDL repository </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ cd hdl/projects/ad9656_fmc/zcu102
+   user@analog:~/hdl/projects/ad9656_fmc/zcu102$ make
+
+The following dropdown contains a table with the parameters that are used to
+configure this project, on :xilinx:`ZCU102`.
```

Comment:
Needs rephrasing, as now there is no collapsible and this is the supported configuration